doc/classes/Control.xml

```diff
@@ -834,58 +838,83 @@
 	</methods>
 	<members>
 		<member name="anchor_bottom" type="float" setter="_set_anchor" getter="get_anchor">
+			Anchors the bottom edge of the node to the origin, the center or the end of its parent container. It changes how the bottom margin updates when the node moves or changes size. Use one of the [code]ANCHOR_*[/code] constants. Default value: [code]ANCHOR_BEGIN[/code].
```

Comment:
Prefer the Oxford comma here (and in the copy and paste similar descriptions):

"the origin, the center, or the end"